CHANGELOG.md

```diff
@@ -47,7 +47,7 @@ This is a list of notable changes to Hyperscan, in reverse chronological order.
 - Bugfix for issue #239: fix hyperscan compile issue under gcc-10.
 - Add VLAN packets processing capability in pcap analysis script. (#214)
 - Avoid extra convert instruction for "Noodle". (#221)
-- Add Hyperscan version marcro in `hs.h`. (#222)
+- Add Hyperscan version macro in `hs.h`. (#222)
```

Comment:
Some projects don't like changing changelogs. Happy to drop this or anything else..